{"fs.defaultFS": "file:///", "hadoop.security.auth_to_local.mechanism": "hadoop", "hadoop.security.groups.cache.secs": "196", "fs.client.resolve.remote.symlinks": "true", "hadoop.service.shutdown.timeout": "202s", "file.stream-buffer-size": "1847151721", "hadoop.security.auth_to_local": "RULE:[1:$1] RULE:[2:$1]", "hadoop.security.groups.cache.background.reload": "false", "hadoop.user.group.static.mapping.overrides": "dr.who=;", "hadoop.kerberos.keytab.login.autorenewal.enabled": "false", "hadoop.security.groups.negative-cache.secs": "112", "fs.creation.parallel.count": "553605745", "file.bytes-per-checksum": "226", "hadoop.security.dns.log-slow-lookups.threshold.ms": "1184286789", "hadoop.security.authentication": "simple", "hadoop.security.dns.log-slow-lookups.enabled": "true", "hadoop.security.groups.cache.background.reload.threads": "31592", "fs.local.block.size": "24251", "fs.file.impl.disable.cache": "true", "io.file.buffer.size": "1112190004", "hadoop.security.token.service.use_ip": "false", "hadoop.security.group.mapping": "org.apache.hadoop.security.JniBasedUnixGroupsMappingWithFallback", "hadoop.security.groups.cache.warn.after.ms": "653"}
org.apache.hadoop.io.file.tfile.TestTFileNoneCodecsByteArrays#testFailureWriterNotClosed